agreements/ddl.sql

```diff
@@ -57,6 +57,25 @@ CREATE TABLE lot_sectors (
 
 CREATE INDEX LOT_SECTORS_IDX1 on LOT_SECTORS (lot_id,sector_code);
 
+CREATE TABLE route_to_market (
+  route_to_market_name              VARCHAR(50) PRIMARY KEY,

```

Comment:
I guess using natural keys rather than surrogate keys is a conscious decision? (just curious as to pros/cons out of interest)

Reply:
There are only going to be four or five records in here . I thought creating a surrogate system generated key was over the top.